model/labels/labels_common.go

```diff
@@ -215,3 +216,7 @@ func contains(s []Label, n string) bool {
 	}
 	return false
 }
+
+func yoloString(b []byte) string {
+	return *((*string)(unsafe.Pointer(&b)))
```

Comment:
Would using string(buf) be more efficient and safer, or are we opting for *((*string)(unsafe.Pointer(&b))) solely to avoid an allocation?

Reply:
We're using the trick to avoid an allocation (and also avoid copying the contents, which would happen if we did `string(buf)`)